astronomer/providers/microsoft/azure/example_dags/example_adf_run_pipeline.py

```diff
@@ -97,17 +95,17 @@ def create_adf_storage_pipeline() -> None:
     storage_string = SecureString(value=CONNECTION_STRING)
 
     ls_azure_storage = LinkedServiceResource(
-        properties=AzureStorageLinkedService(connection_string=storage_string)
+        properties=AzureStorageLinkedService(connection_string=storage_string)  # type: ignore[arg-type]
     )
     adf_client.linked_services.create_or_update(
         RESOURCE_GROUP_NAME, DATAFACTORY_NAME, STORAGE_LINKED_SERVICE_NAME, ls_azure_storage
     )
 
     # Create an Azure blob dataset (input)
-    ds_ls = LinkedServiceReference(reference_name=STORAGE_LINKED_SERVICE_NAME)
+    ds_ls = LinkedServiceReference(type="LinkedServiceReference", reference_name=STORAGE_LINKED_SERVICE_NAME)
```

Comment:
Do we need to support/take care of backward compatibility for the previous versions of the provider?

Reply:
And we used to install azure-mgmt-datafactory >= 1.0.0 here.

Reply:
Also as an example DAG, I think we should use the newer version. WDYT?